### PR TITLE
zdup online migration from openqa ftp source

### PR DESCRIPTION
**Pull request by razghe**
zdup online migration from openqa ftp source to ensure that latest build is used

### CODE REVIEW

File: tests/installation/zdup.pm [hunk 1]
```diff
@@ -40,6 +40,11 @@ sub run() {
     my $defaultrepo;
     if (get_var('SUSEMIRROR')) {
         $defaultrepo = "http://" . get_var("SUSEMIRROR");
+        #SUSEMIRROR not set, zdup from ftp source for online migration
```
Comment (aaannz) [inline tests/installation/zdup.pm]: I don't get the comment. `SUSEMIRROR` is clearly set in this codepath, it passed through if at line 41. Aren't you missing some if-else?

File: tests/installation/zdup.pm [hunk 2]
```diff
@@ -40,6 +40,11 @@ sub run() {
...
+        if (check_var('TEST', "migration_zdup_online_sle12_ga")) {
+            my $online_repo = get_var("REPO_10");
```
Comment (aaannz) [inline tests/installation/zdup.pm]: I think `REPO_10` isn't one of the "standard" vars so I would prefer to have some check if it is actually set.

Comment (lnussel) [inline tests/installation/zdup.pm]: the numbering is completely random, you cannot rely on it

**Comment by razghe** [comment]
I modified it with the correct order .. also tested on my local openqa.
In the vars file(openqa.sused.de) repo_10 is always "REPO_10" : "SLE-12-SP1-Server-DVD-x86_64-Build{number}-Media1". I verified the last 3 builds, and this repo has always been the sle-12-sp1 dvd.

**Comment by lnussel** [comment]
well, if you hardcode REPO_10 I'll go ahead and intentionally randomize REPO_* in the rsync script :-)

**Comment by razghe** [comment]
Ok, I got it :) .. now it will construct the ftp link name from variables, for zypper add repo

**Comment by sysrich** [comment]
Thanks :)

**Comment by sysrich** [comment]
I think this needs work now because of the changes merged from #782

**Pull request merged**